pkg/utils/anonymize/envvars_test.go

```diff
@@ -0,0 +1,47 @@
+package anonymize
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func Test_EnvVar_Obfuscation(t *testing.T) {
+	// Given
+	mock := []corev1.Container{
+		{
+			Env: []corev1.EnvVar{
+				{Name: "NO_TARGET", Value: "original_value"},
+				{Name: "HTTP_PROXY", Value: "original_value"},
+				{Name: "HTTPS_PROXY", Value: "original_value"},
+			},
+		},
+	}
+	envOriginalValue := "original_value"
+
+	// When
+	SensitiveEnvVars(mock)
+
+	// Assert
+	t.Run("Non target env vars keep their original value", func(t *testing.T) {
+		test := mock[0].Env[0]
+		if test.Value != envOriginalValue {
+			t.Logf("\nexpected the variable '%s' to have 'original_value'\ngot: %s", test.Name, test.Value)
+			t.Fail()
```

Comment:
Definitely not a blocker, but we have https://github.com/openshift/insights-operator/blob/master/go.mod#L17 so you can use that library for testing (I think it's more convenient, when we alread have the dependency)